architecture/opp-architecture.adoc

```diff
@@ -15,6 +15,17 @@ toc::[]
 
 include::modules/opp-architecture-architecture.adoc[leveloffset=+1]
 include::modules/opp-architecture-installation.adoc[leveloffset=+1]
+[role="_additional-resources"]
+.Additional resources
+* link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.12/html/installing/installing-preparing#supported-installation-methods-for-different-platforms[Supported installation methods for different platforms]
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.7/html/install/installing[Installing {rh-rhacm}]
+* link:https://github.com/stolostron/policy-collection/blob/main/policygenerator/policy-sets/community/openshift-plus-setup/README.md[Creating and setting up the {ocp} PolicySet]
```

Comment:
Just checking if you got this link to GitHub approved by support? Or perhaps the instructions could be incorporated directly into our docs?